package.json

```diff
@@ -25,6 +25,7 @@
     "nib": "^1.1.0",
     "react": "^15.0.1",
     "react-dom": "^15.0.1",
+    "react-ios-switch": "^0.1.10",
```

Comment:
This adds around 200KB to the JS bundle which is already quite massive (we should investigate that too). I like the robustness of this one such as it's states, draggable etc but it pulls in too many deps and I am not sure we need quite that level of cross-browser support.

We can reuse the one from ShieldDash or create something more robust based on it just not a fan of this NPM package.